### ISSUE TITLE
Project dropdown navigation doesn't respond to keyboard menu controls

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#5807]
## Expected behavior
The project navigation menu should be navigable with the up and down cursor keys.

## Current behavior
After opening the project navigation, with Space or Enter, using the cursor keys scrolls the page up and down.

## Steps to replicate
- Focus the project navigation dropdown on https://www.zooniverse.org/projects/fulsdavid/catalina-outer-solar-system-survey?language=pl
- Open it with Space/Enter.
- Try to navigate the menu with the cursor keys.

## Additional information
Compare the behaviour of the user account menu, which does behave as expected. Focus your name, open the menu with Space/Enter and you can now navigate the menu options with the cursor keys.
